.gitmodules

```diff
@@ -11,5 +11,5 @@
 	branch = oe_port
 [submodule "openenclave"]
 	path = openenclave
-	url = https://github.com/openenclave/openenclave.git
+	url = https://github.com/lsds/openenclave.git
```

Comment:
I don't like having a separate fork of OE.  Is there a reason that these changes can't be pushed to our branch in the OE repo?

Reply:
I've opened PR https://github.com/openenclave/openenclave/pull/3308 to do precisely that. @jhand2 and @mikbras would be good reviewers here.